test/Integration/Dialect/TOSA/i8_neg/dut.cc

```diff
@@ -0,0 +1,13 @@
+void dut(int8_t *restrict v1, int8_t *restrict v2) {
+  v64int8 v3 = broadcast_zero_s8();
+  size_t v4 = 0;
+  size_t v5 = 1024;
+  size_t v6 = 64;
+  for (size_t v7 = v4; v7 < v5; v7 += v6)
+    chess_prepare_for_pipelining chess_loop_range(16, 16) {
+      v64int8 v8 = *(v64int8 *)(v1 + v7);
+      v64int8 v9 = sub(v3, v8);
```

Comment:
From my understanding, the lowering pipeline lowers the integer case of `tosa.negate %arg0` to `zeros - %arg0`. And, we can support those operations with the existing conversion patterns right?

Reply:
Yes, the cycle count is the same as the case using neg intrinsic.